tmt/steps/provision/__init__.py

```diff
@@ -45,6 +45,7 @@
 from tmt.package_managers import FileSystemPath, Package, PackageManagerClass
 from tmt.plugins import PluginRegistry
 from tmt.steps import Action, ActionTask, PhaseQueue
+from tmt.steps.execute.internal import effective_pidfile_root
```

Comment:
This seems to create a circular imports, I recommend moving it into the `setup()` method instead.

Reply:
Done, thanks !